integration/integration_test.go

```diff
@@ -0,0 +1,230 @@
+package integration
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	ps "github.com/mitchellh/go-ps"
+	"github.com/onsi/gomega/gexec"
+	"github.com/stretchr/testify/require"
+
+	"github.com/stretchr/testify/assert"
+)
+
+const TraceeBinaryPath = "../tracee-ebpf/dist/tracee-ebpf"
```

Comment:
another case of go code creating a dependency on the makefile (I called out a similar one in a previous PR), could we manage this better? perhaps an env var that the makefile sets?

Reply:
Updated https://github.com/aquasecurity/tracee/pull/606/commits/3cd02b3b20fc88d6a921bb54ee47b25b60f0d508